gems/sorbet-runtime/lib/sorbet-runtime.rb

```diff
@@ -24,6 +24,8 @@ module T::Private::Types; end
 require_relative 'types/private/class_utils'
 require_relative 'types/private/runtime_levels'
 require_relative 'types/private/methods/_methods'
+require_relative 'types/private/methods/modes'
+require_relative 'types/private/methods/decl_builder'
```

Comment:
ooc, why do these have to move earlier?

Reply:
We need the modes and `DeclBuilder` to exist when we invoke `sig` now, since it returns a builder to permit chaining.